.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,45 @@
+name: Documentation
+description: Report problems and suggest ideas for the documentation
+labels: [docs]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `docs`
+        label](https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3Adocs+is%3Aopen)
+        (including closed issues) before you continue.
```

Comment:
The reason for the line breaks is just for formatting. Otherwise we'd have a very long line. `>-` replaces all newlines with spaces. I disagree with us manually placing line breaks as we do not know what screen size someone is using.

> Also, I think we would allow us to move away from inline links (which look a bit ugly here, I think), could you update that as well?

Good idea, I'll update.

Reply:
I think I misunderstood what you were saying, but I believe `>-` fits our case better anyway.

> for that you need two newlines in a row

Technically, there is a new line, I think you just want two newlines.

`|` will keep all line breaks so we cannot format in YAML and we will have a very long line. `>-` will convert all spaces to line breaks, so we can use line breaks when necessary and still have formatting.

Reply:
Apologies for the confusion - I see now that both `>-` and `|-` fit our use case (I agree that including `-`, aka strip, is better than not including it, aka clip). Because of the way MarkDown is parsed, the additional newlines from `|-` _shouldn't_ matter (though I didn't check whether or not this is actually the case with these forms).